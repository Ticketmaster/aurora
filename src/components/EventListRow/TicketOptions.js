import React from "react";
import styled from "styled-components";

import { colors, typography, spacing } from "../../theme";
import constants from "../../theme/constants";

const TicketOptionsContainer = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-between;

  @media only screen and ${constants.device.mobile} {
    margin-left: ${spacing.giant};
  }
`;

const TicketOptionsTitle = styled.a`
  color: ${colors.pacific.dark};
  font-size: ${typography.size.hecto};
`;

const TicketOptionsSubTitle = styled.div`
  font-size: ${typography.size.centi};
  color: ${colors.sierra.light};
`;

const TicketOptions = () => (
  <TicketOptionsContainer>
    <TicketOptionsTitle>Ticket Options Available</TicketOptionsTitle>
    <TicketOptionsSubTitle>on Partner Site</TicketOptionsSubTitle>
  </TicketOptionsContainer>
);

export default TicketOptions;
