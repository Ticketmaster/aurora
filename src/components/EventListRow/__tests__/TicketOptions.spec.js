import React from "react";
import renderer from "react-test-renderer";

import TicketOptions from "../TicketOptions";

describe("<TicketOptions />", () => {
  it("renders TicketOptions correctly", () => {
    const component = renderer.create(<TicketOptions />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
