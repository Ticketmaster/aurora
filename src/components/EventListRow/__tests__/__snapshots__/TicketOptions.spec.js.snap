// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TicketOptions /> renders TicketOptions correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c1 {
  color: #026cdf;
  font-size: 14px;
}

.c2 {
  font-size: 11px;
  color: #999da0;
}

@media only screen and (min-width:480px) {
  .c0 {
    margin-left: 48px;
  }
}

<div
  className="c0"
>
  <a
    className="c1"
  >
    Ticket Options Available
  </a>
  <div
    className="c2"
  >
    on Partner Site
  </div>
</div>
`;
